DESIGN.md

```diff
@@ -0,0 +1,52 @@
+# Overview
+Warnet is a software suite that allows its users to safely put ideas about Bitcoin to the test. It does this by using Kubernetes to simulate Bitcoin networks and providing user interaction. This document explains how Warnet does this by using design philosophies popular with Kubernetes such as Infrastructure as Code and a tendency towards Stateless Configuration and Externalized Configuration.
+
+# Project structure
+The Warnet code base has four main sections:
+
+1. resources - these items are available during runtime and relate to configuration (crucially, Kubernetes configuration)
+2. src/warnet - python source code lives here
+3. test - CI testing files live here
+4. docs - stores documentation available in the github repository
+
+## Overview of resources
+There are four main kinds of *resources*:
+
+1. Kubernetes configuration files - they are the backbone of Stateless Configuration; they are *yaml* files
```

Comment:
Could be useful to mention here that we prefer helm charts which will be trans-morgified into these kubernetes resources via this [simple process](https://github.com/bitcoin-dev-project/warnet/blob/main/docs/config.md) 😋